website/blog/2018-08-27-7.0.0.md

```diff
@@ -379,6 +379,12 @@ Once `babel-plugin-macros` has been installed and [added to your config](https:/
 
 Learn more about `babel-plugin-macros` in our original post ["Zero-config code transformation with babel-plugin-macros"](https://babeljs.io/blog/2017/09/11/zero-config-with-babel-macros).
 
+### Module Targetting
+
+Babel has always attempted to balance the size impact of transformations and capabilities they provide to JavaScript authors. In Babel 7, it has become much easier to configure Babel to support the growing popularity of the [module/nomodule pattern](https://github.com/kristoferbaxter/preset-env-modules).
+
+Notably, several CLI tools for popular web frameworks ([1](https://cli.vuejs.org/guide/browser-compatibility.html#modern-mode), [2](https://github.com/developit/preact-cli/pull/583)) are already leveraging support leading to a roughly 20% reduction in shipped JavaScript to consumers of applications transpiled by Babel.
```

Comment:
https://cli.vuejs.org/guide/browser-compatibility.html#modern-mode is a todo haha

Reply:
can probably mention meteor too @benjamn